generators/client/templates/angular/webpack/webpack.custom.js.ejs

```diff
@@ -72,6 +71,7 @@ module.exports = async (config, options, targetOptions) => {
   }
 <%_ if (!this.applicationTypeMicroservice) { _%>
   if (targetOptions.target === 'serve' || config.watch) {
+    const tls = config.devServer?.https;
```

Comment:
Optional chaining is only available at node >=14, we still support 12.20.
```suggestion
    const tls = Boolean(config.devServer && config.devServer.https);
```

Reply:
Applied the suggested changes